docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+# Pull Request Template
+
+## Description
+
+:question: Please include a summary of the change(s) a small paragraph in your own words is fine.
+
+
+:question: Break Down of Task, steps taken.
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+:question: Have you used any tutorials or done any reading that helped you to complete these changes and would be of benefit to the group? If yes you can add links here. Great for our growth but also so we can help if needed.
+
+## Issue Addressed
+
+:question: This pull request is in reponse to **issue** #
+
+:question: Does it require the install of any other dependencies apart from the standard node_modules when a **npm install** is run? If yes please list here.
+
+## Type of change
+
+:question: Please tick all relevant options
+
+- [ ] **Bug fix** (non-breaking change which fixes an issue)
+- [ ] **New feature** (non-breaking change which adds functionality)
+- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
+- [ ] **This change requires a documentation update**
```

Comment:
would that go here or in the contributing guidelines?

Reply:
contributing guidelines

Reply:
cool @iesea if you could find space for this in the contributing guidelines #12 #5

Reply:
Hey yes, sorry to sound permeative, although I'll be happy to put these titles into the Guidelines Im unsure if you'd like the whole file to be copied over/ just the titles or a link? Right now I am creating a space for the titles above to go in but please make any changes you would like to see in the doc styling or wording so we have created a good good and clear for all who read it.

Reply:
@AffirmedVisionary and @iesea could you work together to add this suggestion to #12, in a way that works best for the team and is easy for anyone new to jump on and understand 🥰

Reply:
Yes of course would love to look at this further - @AffirmedVisionary let’s work - let me know when you are free and I’ll meet you the same time to bang it out? But mainly to make sure users  can utilise the buttons in the way you vision because I’m unsure myself how to best use it where it clicks

Reply:
@iesea it is hard to keep on top of these reviews/comments in different places. I saw this after reviewing issue #12 and made a suggestion regarding this topic. We carry on conversations regarding this suggestion over there. x

Reply:
> Yes of course would love to look at this further - @AffirmedVisionary let’s work - let me know when you are free and I’ll meet you the same time to bang it out? But mainly to make sure users can utilise the buttons in the way you vision because I’m unsure myself how to best use it where it clicks

Which buttons?